ister_gui.py

```diff
@@ -1608,7 +1608,7 @@ def build_ui_widgets(self):
         if self.timeout:
             wired_req = urwid.Text(('warn', self.timeout))
         else:
-            if self._network_connection():
+            if self.network_connection():
```

Comment:
Why the function rename?

Reply:
@bryteise I renamed it because I need to access that member function on the object; assumed that we were following the leading underscore on private member functions.

Reply:
Ugh sorry, I missed the step.network_connection X(.